release-notes/aws/10.1.0.md

```diff
@@ -0,0 +1,40 @@
+## :zap:  Giant Swarm Release 10.1.0 for AWS is now active for you! :zap:
+
+We are pleased to announce that this new branch (10.1.0) contains the much anticipated Node Pools release and will contain all
```

Comment:
```suggestion
We are pleased to announce that this new release (10.1.0) contains the much anticipated Node Pools feature and will contain all
```

Reply:
Hm. Reading the rest of the sentence shows "branch" is actually better. But do we really need to state that? This is the default everybody expects. There will be other releases after 10.1.0 that will have new features :smile:

Reply:
So maybe we just say: We are pleased to announce that this release contains the much anticipated Node Pools feature.